### PR TITLE
Update CHANGELOG for 0.17.0

### PR DESCRIPTION
**Pull request by testforstephen**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ All notable changes to the "vscode-java-debugger" extension will be documented i
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 0.17.0 - 2019-03-06
+### Added
+- Popup the Exception details via an UI widget when an Exception breakpoint is hit. [#522](https://github.com/Microsoft/vscode-java-debug/issues/522).
```
Comment (yaohaizh) [inline CHANGELOG.md]: Exception ==> exception
an ==> a

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,16 @@ All notable changes to the "vscode-java-debugger" extension will be documented i
...
+
+### Changed
+- Improve the extension load time by using webpack. [#492](https://github.com/Microsoft/vscode-java-debug/issues/492), [#517](https://github.com/Microsoft/vscode-java-debug/issues/517).
```
Comment (yaohaizh) [inline CHANGELOG.md]: Improve ==> Reduce

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,16 @@ All notable changes to the "vscode-java-debugger" extension will be documented i
...
+
+### Fixed
+- Add a new debug configuration `shortenCommandLine` to fix the issue "CreateProcess error=206, The filename or extension is too long". [#110](https://github.com/Microsoft/vscode-java-debug/issues/110).
```
Comment (yaohaizh) [inline CHANGELOG.md]: configuration ==> setting
In VSCode, should use `settings`

**Comment by yaohaizh** [comment]
Please also fix the security issue before release this version of extension.

File: CHANGELOG.md (hunk 3 above)
Comment (testforstephen) [reply CHANGELOG.md]: In launch.json, VSCode called it to configuration. For system settings, called it as user settings.

**Pull request merged**